### PR TITLE
Update keyboard mapping table

### PR DESCRIPTION
**Pull request by ninomp**
* Add shortcut for opening track properties dialog
* Add shortcut for removing/hiding tracks
* Refer to Escape key as `Escape` instead of `ESC`

This is a follow-up to https://github.com/mixxxdj/mixxx/pull/4330 and https://github.com/mixxxdj/mixxx/pull/4347.

### CODE REVIEW

File: source/chapters/appendix/keyboard_mapping_table.rst [hunk 1]
```diff
@@ -132,7 +132,11 @@ language settings the defaults may be different.
 +----------------------------------------+-----------------------------------------+------------------------------------------+
 | Clear search input                     | :kbd:`Ctrl` + :kbd:`Back`                                                          |
 +----------------------------------------+-----------------------------------------+------------------------------------------+
-| Exit search and leave focus            | :kbd:`ESC`                                                                         |
+| Exit search and leave focus            | :kbd:`Escape`                                                                      |
```
Comment (ronso0) [inline source/chapters/appendix/keyboard_mapping_table.rst:135]: while you're at it:
Ctrl + F is a global shortcut set in wtracktableview. cpp, so this should actually be
"Leave current widget and focus tracks table"

Comment (ronso0) [reply source/chapters/appendix/keyboard_mapping_table.rst:135]: Apart from that thos LGTM

Comment (ninomp) [reply source/chapters/appendix/keyboard_mapping_table.rst:135]: > "Leave current widget and focus tracks table"

This text does not fit in this table. Should I (just) enlarge the entire table?

Comment (ronso0) [reply source/chapters/appendix/keyboard_mapping_table.rst:135]: Well, just "Focus tracks table" is sufficient.

We may improve the documentaion later on, for example add `Tab`

**Comment by ronso0** [comment]
LGTM, thank you!

**Pull request merged**